foundation-extensions/target/migration.md

```diff
@@ -0,0 +1,436 @@
+# Migration to AEPTarget
+
+This document is a reference comparison of ACPTarget (2.x) APIs against their equivalent APIs in AEPTarget (3.x) for an iOS mobile application implementation.
+
+If an explanation beyond showing API differences is necessary, it will be captured as a "hint" within that API's section.
+
+For example:
+
+{% hint style="info" %}
+This is information that is important to help clarify the API.
+{% endhint %}
+
+## Pod installation
+
+| AEP (3.x)                    | ACP (2.x) |
+| :----------------------------- | :-------------------------------------- |
+| pod 'AEPTarget', '~&gt; 3.0' | pod 'ACPTarget', '~&gt; 2.0'    |
+| pod 'AEPCore', '~&gt; 3.0'   | pod 'ACPCore', '~&gt; 2.0'              |
+
+## Primary classes
+
+{% tabs %}
+{% tab title="Swift" %}
+
+| AEP (3.x) | ACP (2.x) |
+| :---------- | :-------------------------------------- |
+| Target | ACPTarget                        |
+| MobileCore  | ACPCore                                 |
+
+{% endtab %}
+
+{% tab title="Objective-C" %}
+
+| AEP (3.x)       | ACP (2.x) |
+| :---------------- | :-------------------------------------- |
+| AEPMobileTarget | ACPTarget                        |
+| AEPMobileCore     | ACPCore                                 |
+
+{% endtab %}
+{% endtabs %}
+
+## Primary class
```

Comment:
Why do you have two separate sections, refer to L20 and L42. Shouldn't we merge these? Also I do not see a reference to other public classes e.g. TargetRequest, TargetPrefetch, TargetParameters etc

Reply:
This will be fixed in the migration format change.